content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -362,34 +301,23 @@ anyOf:
 
 Violations of the structural schema rules are reported in the `NonStructural` condition in the CustomResourceDefinition.
 
-Structural schemas are a requirement for `apiextensions.k8s.io/v1`, and disables the following  features for `apiextensions.k8s.io/v1beta1`:
+Structural schemas are required for `apiextensions.k8s.io/v1`. If you use a `apiextensions.k8s.io/v1beta1` CustomResourceDefinitions, that disables the following features:
```

Comment:
Do we have deprecation policy about how long we will wait before removing deprecated API documentation? Since you removed the v1beta1 example in "Create a CustomResourceDefinition", I wonder if we should remove this sentence as well.

Reply:
There's no policy I know of, but that feedback makes sense.

Reply:
The revised text is confusing. 

> that disables the following features

What is "that" referring to? The "structural schema"? or the action of using an old version, or the old CRD?

Reply:
I think you're reviewing superseded commits @tengqm - if I'm wrong, please clarify what should change.